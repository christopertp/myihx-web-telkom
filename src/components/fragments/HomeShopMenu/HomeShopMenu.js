import React from 'react';
import { Link } from 'react-router-dom';
import styles from './styles.scoped.css';

export default function HomeShopMenu() {
  const menu = [
    {
      link: '/shop/internet',
      img: 'ic_shop_internet_white.png',
      alt: 'Internet'
    },
    {
      link: '#',
      img: 'ic_shop_tv_white.png',
      alt: 'TV'
    },
    {
      link: '#',
      img: 'ic_shop_call_white.png',
      alt: 'Call'
    },
    {
      link: '/shop/more',
      img: 'ic_shop_more_white.png',
      alt: 'Other'
    },
  ];

  return (
    <section className={styles.menu}>
      <ul>
        {menu.map((i, idx) =>
          (<li key={idx}>
            <Link
              to={i.link}
            >
              <img
                alt={i.alt}
                src={`/assets/${i.img}`}
              />
              <h5>{i.alt}</h5>
            </Link>
          </li>)
        )}
      </ul>
    </section>
  );
}

