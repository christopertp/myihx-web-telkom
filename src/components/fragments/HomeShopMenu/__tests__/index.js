import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import HomeShopMenu from '../HomeShopMenu';

describe('src/components/fragments/HomeShopMenu', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<HomeShopMenu />);
    expect(tree).toMatchSnapshot();
  });
});
