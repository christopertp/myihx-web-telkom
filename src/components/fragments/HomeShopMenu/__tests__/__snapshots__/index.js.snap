// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/fragments/HomeShopMenu render 1`] = `
<section>
  <ul>
    <li>
      <Link
        to="/shop/internet"
      >
        <img
          alt="Internet"
          src="/assets/ic_shop_internet_white.png"
        />
        <h5>
          Internet
        </h5>
      </Link>
    </li>
    <li>
      <Link
        to="#"
      >
        <img
          alt="TV"
          src="/assets/ic_shop_tv_white.png"
        />
        <h5>
          TV
        </h5>
      </Link>
    </li>
    <li>
      <Link
        to="#"
      >
        <img
          alt="Call"
          src="/assets/ic_shop_call_white.png"
        />
        <h5>
          Call
        </h5>
      </Link>
    </li>
    <li>
      <Link
        to="/shop/more"
      >
        <img
          alt="Other"
          src="/assets/ic_shop_more_white.png"
        />
        <h5>
          Other
        </h5>
      </Link>
    </li>
  </ul>
</section>
`;
